src/main/java/it/pn/frontend/e2e/section/destinatario/personaFisica/HeaderPFSection.java

```diff
@@ -15,6 +15,9 @@ public class HeaderPFSection extends BasePage {
 
     private static final Logger logger = LoggerFactory.getLogger("HeaderPFSection");
 
+    @FindBy(css = "a[title='Sito di PagoPA S.p.A.']")
+    WebElement titleLabel;
+
     @FindBy(css = "button[aria-label='party-menu-button']")
     WebElement profiloUtenteMenu;
```

Comment:
If the **id** or **data-testid** aren't present find elements with `xpath` method